src/platform/web/ui/css/layout.css

```diff
@@ -65,7 +72,7 @@ the layout viewport up without resizing it when the keyboard shows */
 .middle .close-middle { display: none; }
 /* mobile layout */
 @media screen and (max-width: 800px) {
-    .SessionView:not(.middle-shown) {
+    .SessionView:not(.middle-shown, .right-shown) {
```

Comment:
selector lists in `:not` are not supported in IE [apparently](https://developer.mozilla.org/en-US/docs/Web/CSS/:not). Can you write this out?